src/font/font_load_freetype2.cpp

```diff
@@ -256,8 +256,8 @@ static void write_default(void)
 
     if ((pf = OPENFILE(CONFIG_FILE, F("a+")))) {
 
-        fprintf(pf, "[%s]\n", "sung");
-        fprintf(pf, "path=%s\n", "sung.ttf");
+        fprintf(pf, "[%s]\n", "zcoolxiaowei");
```

Comment:
just use "regular" or "default" for font key name

Reply:
Used default.